fabtests/benchmarks/benchmark_shared.c

```diff
@@ -452,9 +453,11 @@ int bandwidth_rma(enum ft_rma_opcodes rma_op, struct fi_rma_iov *remote)
 				ret = ft_post_rma_inject(FT_RMA_WRITE, tx_buf + offset,
 						opts.transfer_size, remote);
 			} else {
-				ret = ft_post_rma(FT_RMA_WRITE, tx_buf + offset,
-						opts.transfer_size, remote,
-						&tx_ctx_arr[j].context);
+				flags = set_fi_more_flag(i, j, flags);
+				ret = ft_post_rma(
```

Comment:
Similar comment as earlier, can we only use FT_RMA_WRITEMSG only when testing fi_more.